### DISCUSSION TITLE
Index closing/deletion and documentation

### DISCUSSION THREAD

**Discussion by abesinger** [Security-Onion-Solutions/securityonion#6074] [Unsupported Versions] [answered]
As a follow-up to #5595, I'd like to suggest that the info about curator close/delete defaults get put more up front in the documentation, at a minimum in the "After Installation" section.  I went through the same things as the poster in #5595, at about the same time, including finding the documentation bugs, but they posted before I could get to it.

I practically had a small heart attack when I first saw logs were missing, as would I expect many people in that situation. Had I been aware of the defaults I probably would have changed them ahead of time.  And yes, I know the documentation does say it, but **where** it is in the documentation makes a difference. Big bold blinking text in the installation section saying "Look at this setting to make sure you don't lose data" would be good.

I'd also suggest having a single global default close/delete setting in addition to setting per-index setting, but I'll post about that separately.

**Comment by dougburks** [comment, marked as answer]
I've updated https://docs.securityonion.net/en/2.3/curator.html and https://docs.securityonion.net/en/2.3/elasticsearch.html and also updated https://docs.securityonion.net/en/2.3/post-installation.html to include links to both of them.

In the future, if you'd like to suggest documentation updates, please feel free to submit pull requests as described at https://docs.securityonion.net/en/2.3/about.html#contributing.

**Reply by abesinger** [reply]
Thanks for pointing that out, I'd be happy to submit PRs for documentation, when time allows. I may also have some code contributions.

And thanks for updating the documentation.
